src/test/kotlin/techcourse/herobeans/integration/GuestControllerIntegrationTest.kt

```diff
@@ -0,0 +1,98 @@
+package techcourse.herobeans.integration
+
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.get
+import techcourse.herobeans.entity.Coffee
+import techcourse.herobeans.entity.Profile
+import techcourse.herobeans.enums.BrewRecommendation
+import techcourse.herobeans.enums.OriginCountry
+import techcourse.herobeans.enums.ProcessingMethod
+import techcourse.herobeans.enums.ProfileLevel
+import techcourse.herobeans.enums.RoastLevel
+import techcourse.herobeans.repository.CoffeeJpaRepository
+
+@SpringBootTest
+@AutoConfigureMockMvc // mockMvc is lighter than SpringBootTest
+class GuestControllerIntegrationTest() {
```

Comment:
```suggestion
@SpringBootTest
@AutoConfigureMockMvc // mockMvc is lighter than SpringBootTest
@ActiveProfiles("test")
class GuestControllerIntegrationTest() {
```

Don't forget to add the profile otherwise the tests won't pass. I will fix it in my next ticket